src/redux/featureToggle/FeatureToggle.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { ReactNode } from 'react';
+import { AppState } from '../reducers';
+import { AsyncDispatch } from '../ThunkTypes';
+import { connect } from 'react-redux';
+import { getFetchFeatureToggleAndDispatchToRedux } from './getFeatureToggleAndDispatchToRedux';
+import LazySpinner from '../../components/LazySpinner';
+
+interface OwnProps {
+    children: ReactNode;
+    toggleID: string;
+    displayWhenFeatureToggleOff?: boolean;
+}
+
+interface StateProps {
+    isOn: boolean;
+}
+
+interface DispatchProps {
+    fetchFeatureToggle: () => void;
+}
+
+type Props = OwnProps & StateProps & DispatchProps;
+
+class FeatureToggle extends React.PureComponent<Props> {
+    componentDidMount() {
+        this.props.fetchFeatureToggle();
+    }
+
+    render() {
+        const pending = this.props.isOn === undefined;
+        if (pending) {
+            return <LazySpinner type="S" delay={1000}/>;
+        }
+        if (!this.props.displayWhenFeatureToggleOff && this.props.isOn) {
```

Comment:
forstår ikke helt disse if-sjekkene.
De viser det samme innholdet dersom
`displayWhenFeatureToggleOff=false && isOn = true` eller `displayWhenFeatureToggleOff=true && isOn = false` 

Skal det returneres null dersom begge er true? for det er jo tilfellet nå. 

Eller er det du egentlig vil sjekke:
`  if (this.props.displayWhenFeatureToggleOff || this.props.isOn)`

Reply:
Mm, skal skrive om denne så den blir mer lesbar 👍